pkg/utils/fileio.go

```diff
@@ -0,0 +1,81 @@
+//
```

Comment:
This is a super thin wrapper of standard functionality. Is it really required?

Reply:
This is required and used for testing, especially when interacting with named pipes. e.g. where operations block until opposing side is connected.
For tests, file i/o is mocked by `MockedFileIO` in `pkg.utils.mocks.io`